Dockerfile

```diff
@@ -0,0 +1,23 @@
+# Используем официальный образ Python
+FROM python:3.8-slim
+
+# Устанавливаем переменные окружения
+ENV PYTHONUNBUFFERED=1
+
+# Создаем рабочую директорию
+WORKDIR /app
+
+# Копируем только requirements.txt для кэширования зависимостей
+COPY requirements.txt /app/
+
+# Устанавливаем зависимости
+RUN pip install --no-cache-dir -r requirements.txt
+
+# Копируем весь проект в контейнер
+COPY . /app/
+
+# Указываем порт, который будет использовать приложение
+EXPOSE 8080
+
+# Команда для запуска приложения
+CMD ["python", "manage.py", "migrate", "&&", "python", "manage.py", "runserver", "0.0.0.0:8080"]
```

Comment:
It's not recommended to include database migration in the CMD instruction. This can lead to issues if the database is not ready when the container starts. Consider handling migrations separately, perhaps in an entrypoint script, to ensure the database is ready before running migrations.